openghg/retrieve/_search.py

```diff
@@ -274,7 +266,7 @@ def local_search(**kwargs):  # type: ignore
 
     # For the time being this will return a dict until we know how best to represent
     # the footprints and emissions results in a SearchResult object
-    if data_type in {"emissions", "footprints", "boundary_conditions", "eulerian_model"}:
+    if data_type in {"column", "emissions", "footprints", "boundary_conditions", "eulerian_model"}:
```

Comment:
Could we use `define_data_types` here as well?

Reply:
This has been added now.